research/auto-fed-rl/src/autofedrl/autofedrl_cifar10_learner.py

```diff
@@ -19,7 +19,6 @@
 import torch
 import torch.nn.functional as F
 import torch.optim as optim
-from pt.learners.cifar10_learner import CIFAR10Learner
```

Comment:
This is marked "deprecated" here, so I just made a copy into the local folder and removed the deprecated line

Reply:
You can still use the original file rather than copying it here.

Reply:
how to still import the deprecated? If I use the original file, it will error out saying class should be code rather than str

Reply:
cannot have the deprecated decorator for inheritance

Reply:
Oh. I thought the deprecation decorator gives only a warning. Using it with class inheritance seems a limitation we cannot easily fix. So, copying the code here should be fine.